docs/tutorials/demo_data_science_project.ipynb

```diff
@@ -9,25 +9,23 @@
    "source": [
     "# Introduction\n",
     "\n",
-    "In this notebook, we will complete a small end-to-end data science tutorial that employs `lakeFS-spec` for data versioning. We will use weather data to train a random forest classifier to predict whether a given day from now is a raining day given the current weather.\n",
+    "In this notebook, we will complete a small end-to-end data science tutorial that employs `lakefs-spec` for data versioning. We will use versioned weather data to train a random forest classifier to predict whether a given day from now is a raining day given the current weather.\n",
     "\n",
     "We will do the following:\n",
     "\n",
     "* Environment setup\n",
-    "* lakeFS setup\n",
-    "* Data ingestion\n",
-    "    * Transactions\n",
-    "    * PUT a file\n",
+    "* LakeFS setup\n",
+    "* Data ingestion via transactions\n",
     "* Model training\n",
     "* Updating data and retraining a model\n",
-    "* Accessing data versions and Reproducing Experiments\n",
-    "* Using a tag instead of a commit SHA for semantic versioning\n",
+    "* Accessing data versions and reproducing experiments\n",
+    "* Using tags for semantic versioning\n",
     "\n",
     "To execute the code in this tutorial as a Jupyter notebook, download this `.ipynb` file to a convenient location on your machine. You can also clone the whole `lakefs-spec` repository. During the execution of this tutorial, in the same directory, a folder `data` will be created. We will also download a file `.lakectl.yaml` to the current user's home directory.\n",
     "\n",
     "This tutorial assumes that you have installed `lakefs-spec` in a virtual environment, and that you have followed the [quickstart guide](../quickstart.md) to set up a local lakeFS instance.\n",
     "\n",
-    "# Environment Setup\n",
+    "## Environment setup\n",
     "\n",
     "Install the libraries necessary for this notebook on the environment you have just created: "
    ]
```

Comment:
I propose to change:  "%pip install lakefs-spec numpy pandas scikit-learn"
To: "%pip install notebook numpy pandas scikit-learn"
Argumentation: lakefs-spec already installed in the quickstart tutorial; assuming a user created vanilla fresh venv, it requires notebook installation.

Reply:
This cell is a Jupyter magic instruction, so it will not run if `notebook` is not already installed.

Reply:
Currently, this cell execution breaks the notebook as it pulls the latest release of `lakefs-spec` from PyPI. In this release, we do not have e.g. the `create_repository` function. 

This particular problem should be fixed with a new PyPI release. But in general, I would prefer the installation to point to the current version on `main` as to not run into a problem due to the divergence of the GitHub Repo and PyPI again.